From 884ed3e7decbb91a436f1f8b65c72a1abc9f41ec Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 18 Jan 2024 09:31:34 +0100
Subject: [PATCH] com.webos.service.systemservice: Add image.management to
 groups file.

Fixes:

WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: Found 34 group(s) that appear only in api-permissions.d, consider define them in groups.d
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg: === LIST BEGIN: Groups used in api-permissions.d but not defined in groups.d ===
WARNING: luneos-dev-image-1.0-r0 do_validate_ls2_acg:   image.management

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [LuneOS specific]

 files/sysbus/com.webos.service.systemservice.groups.json | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/files/sysbus/com.webos.service.systemservice.groups.json b/files/sysbus/com.webos.service.systemservice.groups.json
index 290c984..95b7cb1 100644
--- a/files/sysbus/com.webos.service.systemservice.groups.json
+++ b/files/sysbus/com.webos.service.systemservice.groups.json
@@ -4,5 +4,6 @@
     "systemsettings.query" : ["oem"],
     "software.query" : ["dev"],
     "time.management" : ["oem"],
-    "time.query" : ["dev"]
+    "time.query" : ["dev"],
+    "image.management" : ["dev"]
 }
